setup.py

```diff
@@ -18,7 +19,7 @@
 here = Path().resolve()
 
 # Get the long description from the README file
-with open((here / "README.md"), encoding="utf-8") as f:
+with open(os.path.join(here, "README.md"), encoding="utf-8") as f:
```

Comment:
I think this needs to be changed back. Here is a path object and so the "/" operator gets overloaded